nextflow_schema.json

```diff
@@ -222,25 +230,61 @@
                     "default": "'1000000,500000'",
                     "description": "Resolution to build the maps (comma separated)"
                 },
-                "ice_filer_low_count_perc": {
-                    "type": "string",
+                "hicpro_maps": {
+                    "type": "boolean",
+                    "description": "Generate raw and normalized contact maps with HiC-Pro"
+                },
+                "ice_filter_low_count_perc": {
+                    "type": "number",
                     "default": 0.02,
-                    "description": "Filter low counts rows before normalization"
+                    "description": "Filter low counts rows before HiC-Pro normalization"
                 },
-                "ice_filer_high_count_perc": {
+                "ice_filter_high_count_perc": {
                     "type": "integer",
-                    "default": "0",
-                    "description": "Filter high counts rows before normalization"
+                    "description": "Filter high counts rows before HiC-Pro normalization"
                 },
                 "ice_eps": {
-                    "type": "string",
-                    "default": "0.1",
-                    "description": "Threshold for ICE convergence"
+                    "type": "number",
+                    "default": 0.1,
+                    "description": "Threshold for HiC-Pro ICE convergence"
                 },
                 "ice_max_iter": {
                     "type": "integer",
-                    "default": "100",
-                    "description": "Maximum number of iteraction for ICE normalization"
+                    "default": 100,
+                    "description": "Maximum number of iteraction for HiC-Pro ICE normalization"
+                },
+                "res_zoomify": {
+                    "type": "string",
+                    "default": "5000",
```

Comment:
```suggestion
                    "type": "number",
                    "default": 5000,
```

Reply:
or could this be also an array like `res_tads`? If you want to be extra careful here, you could write a regex pattern for these arrays, to be sure to only get comma-separated numbers.

Reply:
Yes, all `res_XXX` parameters are list of values separated with a comma. So I should keep the `string` type.
Regarding the regexp, would you have any exemple ?

Reply:
the following should work `^(\d+)(,\d+)*$`, tested here: https://regex101.com/r/Wfll3P/2

Reply:
Sorry I wasn't clear, do you mean that I can put a regexp for the type ?
```
"type"="^(\d+)(,\d+)*$"
```

Reply:
pattern added for all options expected a list of comma separated value in the json file